modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/resources/manifest/ExportedArtifactsCollectorTestCase.java

```diff
@@ -74,8 +74,8 @@ public void setup() {
   @Test
   public void collect() {
     Set<String> exportedPackages = collector.getExportedPackages();
-    assertThat(exportedPackages, hasSize(3));
-    assertThat(exportedPackages, hasItems(SHAPE_PACKAGE, APPLE_PACKAGE, VEGAN_PACKAGE));
+    assertThat(exportedPackages, hasSize(4));
+    assertThat(exportedPackages, hasItems(SHAPE_PACKAGE, APPLE_PACKAGE, VEGAN_PACKAGE, PEEL_PACKAGE));
```

Comment:
add another test case considering composite cases, such as:

* List<List<Pojo>>
* List<Map<String, Pojo>>
*  Pojo which has a field of another pojo